### ISSUE TITLE
Uninstall - "not installed"

### ISSUE THREAD

**Issue by dylan-chong** [kylef/swiftenv#106]
```
> swiftenv versions
  3.0
  4.0.2

> swiftenv uninstall 3.0
3.0 is not installed.
```

**Comment by dylan-chong** [comment]
This was probably caused by not following the [installation instructions](https://swiftenv.fuller.li/en/latest/installation.html) properly (i did not set the environment variables)

**Comment by peterlobster** [comment]
I'm having this game problem and I have my environment variables set to (I installed via Homebrew):
```
export SWIFTENV_ROOT="/usr/local/var/swiftenv"
export PATH="$SWIFTENV_ROOT/bin:$PATH"
eval "$(swiftenv init -)"
if which swiftenv > /dev/null; then eval "$(swiftenv init -)"; fi
```

```
$ swiftenv versions
  3.1.1
* 4.1 (set by /usr/local/var/swiftenv/version)
```
```
$ swiftenv uninstall 3.1.1
3.1.1 is not installed.
```
```
swift -v
Apple Swift version 3.1 (swift-3.1.1-RELEASE)
Target: x86_64-apple-macosx10.9
/Library/Developer/Toolchains/swift-3.1.1-RELEASE.xctoolchain/usr/bin/lldb "--repl=-enable-objc-interop -sdk /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.13.sdk -color-diagnostics"
Welcome to Apple Swift version 3.1 (swift-3.1.1-RELEASE). Type :help for assistance.
  1>  

```

**Comment by peterlobster** [comment]
@kylef Could you please reopen this ticket?

Also I think that the documentation is a bit unclear as to what you need to add your your .bash_profile. For example, I installed via Homebrew, and I wasn't clear about haven't to use the environment variables. I might still have the wrong ones...

**Comment by kylef** [comment]
> ```bash
> export SWIFTENV_ROOT="/usr/local/var/swiftenv"
> export PATH="$SWIFTENV_ROOT/bin:$PATH"
> eval "$(swiftenv init -)"
> if which swiftenv > /dev/null; then eval "$(swiftenv init -)"; fi
> ```

Line 2 here isn't needed if you're using homebrew, and line 3 is redundant as line 4 does the same thing effectively (providing swiftenv command is found).

So in your profile all you will need:

```bash
export SWIFTENV_ROOT="/usr/local/var/swiftenv"
if which swiftenv > /dev/null; then eval "$(swiftenv init -)"; fi
```

---

I'm able to reproduce your problem and seems there is a bug uninstalling toolchains. This toolchain has a `-RELEASE` suffix in a path that was unexpected. I've fixed the bug in 194e02b.

If you'd like to upgrade to current master to use this fix you can via running the following commands:

```shell
$ brew unlink swiftenv
$ brew install kylef/formulae/swiftenv --HEAD
$ swiftenv uninstall 3.1.1
```

**Comment by peterlobster** [comment]
@kylef 

Thanks for clarifying!

Also I updated and I can confirm everything is working like a glove!

Thanks for all the help!
